subprojects/docs/src/snippets/tasks/incrementalBuild-incrementalBuildAdvanced/kotlin/buildSrc/src/main/java/org/example/GitClone.java

```diff
@@ -33,7 +33,7 @@
 import java.io.IOException;
 
 // tag::git-clone[]
-@UntrackedTask(because = "Git tracks the state")                                           // <1>
```

Comment:
:?: Those indents are intentional, so the callouts align in the docs. Do you think that is a bad idea?

Reply:
We don't align these number labels anywhere else in the docs as far as I've seen. In this particular case it wasn't immediately clear to me which line the (1) is referring to.

Overall, I think the contrast of this labels is high enough, that they do not mix with the code and putting them close makes the experience better. This works especially well, when the snippets are followed by a list that has the same numbered labels instead of bullets.

![image](https://user-images.githubusercontent.com/2759152/208418510-84f0f3c8-5fc1-4765-87f1-186e70a01c63.png)

Reply:
Good for me.